mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionFilter.java

```diff
@@ -121,16 +129,19 @@ public String buildCriteria() {
       addCondition(conditionBuilder, buildInCondition(UI_STATUS_FIELD, uiStatuses));
     }
     if (isNotEmpty(hrIdPattern) && isNotEmpty(fileNamePattern)) {
-      conditionBuilder.append(String.format(" AND (%s OR %s)", buildCaseSensitiveLikeCondition(HRID_FIELD, hrIdPattern),
-        buildCaseInsensitiveLikeCondition(FILE_NAME_FIELD, fileNamePattern)));
+      conditionBuilder.append(String.format(" AND (%s OR %s)", buildCaseSensitiveLikeCondition(HRID_FIELD, hrIdPattern, false),
+        buildCaseInsensitiveLikeCondition(FILE_NAME_FIELD, fileNamePattern, false)));
     } else {
       if (isNotEmpty(hrIdPattern)) {
-        addCondition(conditionBuilder, buildCaseSensitiveLikeCondition(HRID_FIELD, hrIdPattern));
+        addCondition(conditionBuilder, buildCaseSensitiveLikeCondition(HRID_FIELD, hrIdPattern, false));
       }
       if (isNotEmpty(fileNamePattern)) {
-        addCondition(conditionBuilder, buildCaseInsensitiveLikeCondition(FILE_NAME_FIELD, fileNamePattern));
+        addCondition(conditionBuilder, buildCaseInsensitiveLikeCondition(FILE_NAME_FIELD, fileNamePattern, false));
       }
     }
+    if (isNotEmpty(excludeJobProfileNames)) {
+      addCondition(conditionBuilder, buildCaseInsensitiveLikeCondition(FILE_NAME_FIELD, excludeJobProfileNames, true));
```

Comment:
```suggestion
      addCondition(conditionBuilder, buildCaseInsensitiveLikeCondition(JOB_PROFILE_NAME_FIELD, excludeJobProfileNames, true));
```
please, update this line with JOB_PROFILE_NAME_FIELD because here we must filter out jobs by job profile name instead of file name

Reply:
thanks!